docs/track.rst

```diff
@@ -447,6 +447,8 @@ This example requires that the ``ingest-geoip`` Elasticsearch plugin is installe
 
 This is an administrative operation. Metrics are not reported by default. Reporting can be forced by setting ``include-in-reporting`` to ``true``.
 
+Please see `the pipeline documentation <https://www.elastic.co/guide/en/elasticsearch/reference/current/handling-failure-in-pipelines.html>`_ for details on handling failures in pipelines.
```

Comment:
Maybe move this paragraph so it is directly below the example? Also, some introductory text would be nice; something along the lines of: 

> In the following example we setup a pipeline that ... . Note that we need to use `raw` / `endraw` texts to ensure that Rally passes the Mustache template as is and does not attempt to resolve it itself ...

(and then link back to your newly added note about templating Rally?)

Reply:
done